fe/fe-core/src/main/java/com/starrocks/connector/jdbc/JDBCMetadata.java

```diff
@@ -67,13 +67,22 @@ public JDBCMetadata(Map<String, String> properties, String catalogName) {
             LOG.warn("{} not support yet", properties.get(JDBCResource.DRIVER_CLASS));
             throw new StarRocksConnectorException(properties.get(JDBCResource.DRIVER_CLASS) + " not support yet");
         }
+        schemaResolver.setSupportPartitionInformation(checkSupportPartitionInformation());
```

Comment:
Do you think merge checkAndSet together will be more clear?

Reply:
This writing method is for the convenience of unit testing. I'll optimize this place.